packages/fcl-core/src/interaction-template-utils/get-template-argument-message.js

```diff
@@ -2,15 +2,15 @@ import {invariant} from "@onflow/sdk"
 
 /**
  * @description Gets Interaction Template argument message by message key, argument label, and localization
- * 
+ *
  * @param {object} opts
  * @param {string} opts.localization [localization="en-US"] - Localization to get message for
  * @param {string} opts.argumentLabel - Argument label to get message for
  * @param {string} opts.messageKey - Message key to get message for
  * @param {object} opts.template - Interaction Template to get message from
  * @returns {string} - Message
  */
-export function getTemplateArgumentMessage({
+export async function getTemplateArgumentMessage({
```

Comment:
this function is sync
```suggestion
export function getTemplateArgumentMessage({
```